Initial draft of the 'Stay on target' lesson (#59)
carpentries/lesson-development-training: episodes/09-formative-assessment.md

::::::::::::::::::::::::::::::::::::::::::::::::::

As we have seen previously, defining objectives for a lesson (or a teaching episode) can help to focus your content on the most important learning outcomes and outline the scope of your lesson project. With learning objectives you have defined your **intended (or planned) curriculum** - what you set out to accomplish. What skills and knowledge your learners actually leave with and can demonstrate after the workshop - the **attained curriculum** - will be a combination of **implemented** and **hidden** curricula:

>   - **implemented curriculum** are the concepts, relationships and skills that are explicitly covered in your lesson and translated into classroom practices - i.e. the teaching & learning activities,

Reviewer: ```suggestion
  - **implemented curriculum** is the concepts, relationships and skills that are explicitly covered in your lesson and translated into classroom practices - i.e. the teaching & learning activities,
```
curriculum/curricula terms are difficult to use the right matching singluar/plural verb...

::::::::::::::::::::::::::::::::::::::  callout

## Useful reading

Useful further reading: [Course design: Considerations for Trainers – a Professional Guide](https://f1000research.com/documents/9-1377).

::::::::::::::::::::::::::::::::::::::::::::::::::

## Assessements

In order to measure progress and evaluate if and what learning occurred - we use various types of assessments:

 - **formative assessments** - used to detect changes in learner performance *during*
instruction, to provide feedback and insight into the learners' developing mental models of the topic taught and to identify any old or developing misconceptions. 
- **summative assessments** - used to verify whether learners achieved
the stated learning objectives after instruction.

Summative assessments sum up what learning has been achieved after 
training (e.g. via exams). They give valuable
data about learning attainment by individuals and entire cohorts but are not used to guide further progress. They may not be as suitable for short courses, but may be necessary for those that give marks or certificates of completion. 

Formative assessments are applied throughout a course; they

> can inform decisions about how to modify instruction to better promote learning; they can also inform learners about changes they may need to make to improve their learning.

Reviewer 2: It would be good to mention formative assessment as a way to move information from working memory to long-term memory here too, linking to the relevant section of the Instructor Training curriculum https://carpentries.github.io/instructor-training/05-memory/index.html#using-formative-assessment-to-support-memory-consolidation

Author: Fixed via 0de9ec6 - check if this is better now @tobyhodges.

the awareness a learner has that they are succeeding in learning something new. 
Any instructional tool that generates feedback and is used in a formative way to check for learners' understanding can be described as “formative assessment”. For example, 

- reflection at the end of a session to help process learning - e.g. asking learners to write down things they learned, things they want to know more about and any questions they still have
- concept maps - asking learners to reflect by drawing a diagram or writing down a list of new concepts and skills they’ve learned and how they connect with previous knowledge
- minute cards - gauging learners’ satisfaction and understanding using agreed signals (e.g. raising different coloured post-it notes to indicate that the pace is too fast/slow, that they completed/have not completed an exercise).

> Many other formative assessment tools can be found in Briggs’ list of ["21 ways to check for student understanding"](ww.opencolleges.edu.au/informed/features/21-way).

Reviewer 2: ```suggestion
Many other formative assessment tools can be found in Briggs’ list of ["21 ways to check for student understanding"](https://www.opencolleges.edu.au/informed/features/21-ways-to-check-for-student-understanding/).
```

::::::::::::::::::::::::::::::::::::::::::::::::::

[Multiple choice question (MCQ)](https://carpentries.github.io/instructor-training/02-practice-learning/#using-formative-assessment-to-identify-misconceptions) exercises are types of a formative assessment that can help you target anticipated misconceptions. When designed carrefully, each incorrect answer in a MCQ will have diagnostic power and provide valuable insight into how a mental model is broken. For example, suppose we are teaching children multi-digit addition. An example of a well-designed MCQ ([borrowed from The Carpentries Instructor Training](https://carpentries.github.io/instructor-training/02-practice-learning/#using-formative-assessment-to-identify-misconceptions)) in this case could be: 

> Q: What is 27 + 15?
> 
> a) 42
> b) 32
> c) 312
> d) 33

Reviewer: ```suggestion
> Q: What is 27 + 15?
> 
> a) 42
> b) 32
> c) 312
> d) 33
```
Can this still be in a block quote?  Make it stand out a little.

Author: Yes! I was not sure how that is done in the new lesson infrastructure and I was just rushing to submit the first draft. Thank you @sstevens2.

The correct answer is 42, but each of the other answers provides a valuable insight:

> b) they do not understand the concept of a carry and are throwing it away completely
> c) understand the concept of a carry and know that they cannot just discard the carried ‘1’, but do not understand that it is actually a ten and needs to be added into the next column - they are treating each column of numbers as unconnected to its neighbours.
> d) they understand that they need to carry ‘1’ but are adding it to the wrong column

Reviewer: ```suggestion
> b) they do not understand the concept of a carry and are throwing it away completely
> c) understand the concept of a carry and know that they cannot just discard the carried ‘1’, but do not understand that it is actually a ten and needs to be added into the next column - they are treating each column of numbers as unconnected to its neighbours.
> d) they understand that they need to carry ‘1’ but are adding it to the wrong column
```
